lenovo/thinkpad/x230/default.nix

```diff
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, ... }:
+
+with lib; {
+  imports = [
+    ../.
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop/hdd
+  ];
+
+  boot = {
+    kernelModules = [
+      "kvm-intel"
+      "acpi_call"
+      "tpm-rng"
+    ];
+    extraModulePackages = with config.boot.kernelPackages; [
+      acpi_call
+    ];
+  };
+
+  hardware.opengl.extraPackages = with pkgs; [
+    vaapiIntel
+    vaapiVdpau
+    libvdpau-va-gl
```

Comment:
I always assumed this is on with `services.xserver.videoDrivers = [ "intel" ];`, but seems it's not :-( Do you think it's safe to enable it for other computers with modern Intel integrated graphics?

Reply:
I also had to double-check it. Enabling it greatly increases battery life time when watching videos.
Regarding other computers, I have no computers to test unfortunately.

For checking if vdpau and va is enabled you can run `nix-shell -p libva.bin vdpauinfo --run "vainfo; vdpauinfo"`

Reply:
This is a very good find, and possibly all nixos-hardware users now will have better battery life. Thank you a lot!

I'm going to test this on a few machines that I have that are not from Lenovo...